src/Umbraco.Core/PropertyEditors/ValueConverters/CheckboxListValueConverter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Models.PublishedContent;
+
+namespace Umbraco.Core.PropertyEditors.ValueConverters
+{
+    public class CheckboxListValueConverter : PropertyValueConverterBase, IPropertyValueConverterMeta
```

Comment:
Since these will all be default property value converters, they need the DefaultPropertyValueConverterAttribute attribute